package.json

```diff
@@ -8,6 +8,6 @@
   },
   "scripts": {
     "start": "gulp clean && gulp compile && gulp theme",
-    "postinstall": "rm -rf pattern-lab && composer create-project -n pattern-lab/edition-twig-standard pattern-lab && rm -rf pattern-lab/source && ln -s ../components pattern-lab/source"
+    "postinstall": "rm -rf pattern-lab && composer create-project -n pattern-lab/edition-twig-standard pattern-lab && rm -rf pattern-lab/source && ln -s ../components pattern-lab/source && git clone https://github.com/drupal-pattern-lab/bem-twig-extension.git && cp bem-twig-extension/bem.function.php components/_twig-components/functions && rm -rf bem-twig-extension/"
```

Comment:
You may want to consider writting a small node script for this using https://www.npmjs.com/package/fs-extra and https://www.npmjs.com/package/cross-spawn so that emulsify will work on windows machines. If you want help with this, I can carve out some time!